x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeResponse.java

```diff
@@ -40,14 +41,16 @@ final class ComputeResponse extends TransportResponse {
         Integer totalShards,
         Integer successfulShards,
         Integer skippedShards,
-        Integer failedShards
+        Integer failedShards,
+        boolean isPartial
```

Comment:
Since this requires a TransportVersion change to add, I wonder if we should have @luigidellaquila review this as well to see if it is consistent with the general purpose allow_partial results?